File: __tests__/__fixtures__/output/python/python3/full.py
import http.client

conn = http.client.HTTPSConnection("httpbin.org")

payload = "foo=bar"

headers = {
    'cookie': "foo=bar; bar=baz",
    'accept': "application/json",
    'content-type': "application/x-www-form-urlencoded"
    }

conn.request("POST", "/anything?foo=bar&foo=baz&baz=abc&key=value", payload, headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))

File: src/targets/javascript/jquery/fixtures/full.js
const settings = {
  async: true,
  crossDomain: true,
  url: 'https://httpbin.org/anything?foo=bar&foo=baz&baz=abc&key=value',
  method: 'POST',
  headers: {
    cookie: 'foo=bar; bar=baz',
    accept: 'application/json',
    'content-type': 'application/x-www-form-urlencoded'
  },
  data: {
    foo: 'bar'
  }
};

$.ajax(settings).done(res => {
  console.log(res);
});